core-services/ci-chat-bot/README.md

```diff
@@ -0,0 +1 @@
+This folder contains extra configuration for the ci-chat-bot (a.k.a. cluster-bot) service.
```

Comment:
Would you mind expanding this some to explain what's supported in this file?  Potentially, something includes a pointer to the type definition over in the ci-chat-bot code and maybe a blurb about reaching out in `#forum-crt` with questions.

Reply:
Updated